draft-packages/filter-drawer/KaizenDraft/FilterDrawer/FilterDrawer.tsx

```diff
@@ -0,0 +1,98 @@
+import { MenuContent } from "@kaizen/draft-menu"
+import { StatelessMenu } from "@kaizen/draft-menu/KaizenDraft/Menu/StatelessMenu"
+import classnames from "classnames"
+import * as React from "react"
+const filterIcon = require("@kaizen/component-library/icons/filter.icon.svg")
+  .default
+const styles = require("./styles.module.scss")
+import { FilterDrawerButton } from "./FilterDrawerButton"
+
+export interface FilterDrawerProps {
+  /**
+   * The text that goes inside the filter button
+   */
+  labelText: string
+
+  /**
+   * Metadata shows relevant info about the content being filtered
+   */
+  metadata?: React.ReactNode
```

Comment:
Nit/ I wasn't quite sure what metadata referred to on first read through. It wasn't until I looked at the story it clicked that this is the stuff that displays beside the filter drawer. That being said, I can't think of a better name 🤷

Reply:
I got the name from the Figma, and I think it makes sense to use the same anme.

Reply:
That sounds reasonable to me